plugin.xml

```diff
@@ -51,14 +53,25 @@ xmlns:android="http://schemas.android.com/apk/res/android"
         </config-file>
 
         <source-file src="src/android/Geolocation.java" target-dir="src/org/apache/cordova/geolocation/" />
+        <source-file src="src/android/LocationContext.java" target-dir="src/org/apache/cordova/geolocation/" />
+        <source-file src="src/android/LocationUtils.java" target-dir="src/org/apache/cordova/geolocation/" />
+        <source-file src="src/android/OnLocationResultEventListener.java" target-dir="src/org/apache/cordova/geolocation/" />
+        <source-file src="src/android/LocationError.java" target-dir="src/org/apache/cordova/geolocation/" />
 
-        <js-module src="www/android/geolocation.js" name="geolocation">
-            <clobbers target="navigator.geolocation" />
+        <js-module src="www/Coordinates.js" name="Coordinates">
+            <clobbers target="Coordinates" />
```

Comment:
Are all these clobbers required?

Reply:
They exist on iOS and it seems it's a design decision from the original author in order to expose those objects to anyone willing to use them (see [here](https://github.com/OutSystems/cordova-plugin-geolocation/blob/outsystems/README.md#objects-read-only))